stories/login/pages/Terms.stories.tsx

```diff
@@ -45,29 +45,44 @@ export const French: Story = {
         />
     )
 };
-export const WithErrorMessage: Story = {
+
+export const Spanish: Story = {
     render: () => (
         <KcPageStory
             kcContext={{
-                messagesPerField: {
-                    existsError: () => true,
-                    get: () => "An error occurred while processing your request."
+                locale: {
+                    currentLanguageTag: "es"
+                },
+                "x-keycloakify": {
+                    messages: {
+                        termsText: "<p>Mis términos en <strong>Español</strong></p>"
+                    }
                 }
             }}
         />
     )
 };
 
-export const Spanish: Story = {
+export const LongMessage: Story = {
     render: () => (
         <KcPageStory
             kcContext={{
                 locale: {
-                    currentLanguageTag: "es"
+                    currentLanguageTag: "en"
```

Comment:
There's no need to specify the language here since english is the default.  
Let's just specify th properties that diverges from the defaults.